apps/chat/src/pages/api/themes/styles.ts

```diff
@@ -103,6 +119,7 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
         wrapCssContents(`.${theme.id}`, [
           generateColorsCssVariables(theme.colors),
           generateUrlsCssVariables({ 'app-logo': theme['app-logo'] }),
+          generateFontCssVariables(theme.fontFamily),
```

Comment:
I think it should work as previous line 
`generateFontCssVariables('theme-font', theme['font-family']`
i.e. just string should be provided in theme

Reply:
Changed as suggested.
Function `generateFontCssVariables` changed accordingly